15LessonFifteen/Checkpoint3/css/style.css

```diff
@@ -0,0 +1,160 @@
+main>section:first-child {
+  background: gray url('https://gualerdz57.github.io/intro-workbook/15LessonFifteen/Checkpoint3/img/wine-bg.jpg') no-repeat center center;
+  background-blend-mode: multiply;
+  background-size:cover;
+}
+.border-around{
+
+  border-left:1px solid white;
+}
+.flex-container{
+  display:flex;
+}
+
+.column-flow{
+  flex-flow:column wrap;
+}
+
+.row-flow{
+  flex-flow:row wrap;
+}
+
+.box-2{
+  width:100%;
+}
+
+.box-3{
+  width:75%;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+.flex-center{
+  justify-content:center;
+  align-items:center;
+  text-align:center;
+}
+
+.famous-quote{
+  text-align:center;
+  font-style: italic;
+}
+
+.white-text{
+  color:#fff;
+}
+
+.wine-background{
+  background: url('https://gualerdz57.github.io/intro-workbook/15LessonFifteen/Checkpoint3/img/wine16.jpg') no-repeat center center;
+  background-size:cover;
+  height:300px;
+}
+
+.vine-background{
+  background: url('https://gualerdz57.github.io/intro-workbook/15LessonFifteen/Checkpoint3/img/wine13.jpg');
+  background-size:cover;
+  height:500px;
+}
+.table-background{
+  background: url('http://foundry.mediumra.re/img/wine12.jpg') no-repeat center center;
+  background-size:cover;
+  height:500px;
+  color:#fff;
+}
+.table-background div{
+  width:70%;
+}
+.table-background h2{
+  width:100%;
+  text-align:center;
+}
+.table-background img{
+  width:100px;
+}
+.navbar-default{
+    background-color: transparent;
```

Comment:
Looks _great_ when at the top, since the background is dark. Not ideal as the user scrolls down over lighter content... makes it unreadable.